presentation/src/main/java/com/threegap/bitnagil/presentation/emotion/EmotionScreen.kt

```diff
@@ -0,0 +1,135 @@
+package com.threegap.bitnagil.presentation.emotion
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.threegap.bitnagil.designsystem.BitnagilTheme
+import com.threegap.bitnagil.presentation.common.flow.collectAsEffect
+import com.threegap.bitnagil.presentation.emotion.model.Emotion
+import com.threegap.bitnagil.presentation.emotion.model.mvi.EmotionSideEffect
+import com.threegap.bitnagil.presentation.emotion.model.mvi.EmotionState
+
+@Composable
+fun EmotionScreenContainer(
+    viewModel: EmotionViewModel = hiltViewModel(),
+    navigateToBack: () -> Unit,
+) {
+    val state by viewModel.stateFlow.collectAsState()
```

Comment:
[2] 추후 해당 방향도 좋을거 같습니다요~!
```suggestion
    val state by viewModel.stateFlow.collectAsStateWithLifecycle()
```